script.trakt/tests/fixtures/compare_shows_batman_episode_to_add.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
please remove the tests/ folder from submission. Maybe you can think about automating the submissions and add all those files to .gitattributes

Reply:
Done